apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/container/tab.js

```diff
@@ -1,20 +1,14 @@
 /**
- *
```

Comment:
Not sure where this changes comes from... when I run `yarn`, the file got changed automatically 🤷

Reply:
Maybe an updated lint rule dealing with jsdoc?